src/pages/WavmanPlayer.tsx

```diff
@@ -170,8 +170,14 @@ const WavmanPlayer: React.FC<{
   }, [kind1NowPlaying, relay]);
 
   const [kind32123NowPlaying] = kind32123Events || [];
+  const defaultTrackInfo: Partial<WavlakeEventContent> = {
+    title: "",
+    enclosure: "",
+    creator: "",
+    link: "",
```

Comment:
we can alternatively add a `Loading` text here so it shows up in the player

Reply:
we may want to force multi line title/artist for everyone, else the music notes jump around while loading.

I'd prefer everything to have static placement on the screen, no shifting around